### PR TITLE
Adding youtube video for custom plugin

### PR DESCRIPTION
**Pull request by irishtek-solutions**
This is for validating video on how to add a custom plugin

### CODE REVIEW

**Review by DeclanPaulKeane — COMMENTED** [review]
Looks good!

**Review by DeclanPaulKeane — APPROVED** [review]
Reviewed video, looks good.

**Review by DeclanPaulKeane — APPROVED** [review]

**Review by DeclanPaulKeane — APPROVED** [review]
Good

**Comment by DeclanPaulKeane** [comment]
Done

File: plugins/custom-plugins/calling-external-auth-endpoint/README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 ## Video for adding external Authentication Endpoint Custom Plugin in Konnect
 
+Click on [youtube link](https://youtu.be/utqH-zGzReY?si=zKmrv-WabErynC2j) to view how this custom plugin works.
```
Comment (DeclanPaulKeane) [inline plugins/custom-plugins/calling-external-auth-endpoint/README.md:5]: Great

**Review by DeclanPaulKeane — APPROVED** [review]

**Review by DeclanPaulKeane — CHANGES_REQUESTED** [review]
Change this

**Review by irishtek-solutions — COMMENTED** [review]
Thanks

**Review by irishtek-solutions — COMMENTED** [review]
Done

**Pull request merged**